lighthouse-cli/test/fixtures/a11y/a11y_tester.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
is this just for local testing right now? Is the plan to make something like the dbw smoke tests?